File: C# projects/projects/Anagram_Checker/Anagram_Checker.cs
using System;

namespace AnagramChecker
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Enter the first string:");
            string str1 = Console.ReadLine();

            Console.WriteLine("Enter the second string:");
            string str2 = Console.ReadLine();

            if (AreAnagrams(str1, str2))
            {
                Console.WriteLine("The strings are anagrams.");
            }
            else
            {
                Console.WriteLine("The strings are not anagrams.");
            }
        }

        static bool AreAnagrams(string str1, string str2)
        {
            char[] arr1 = str1.ToCharArray();
            char[] arr2 = str2.ToCharArray();
            
            Array.Sort(arr1);
            Array.Sort(arr2);
            
            return new string(arr1) == new string(arr2);
        }
    }
}

File: C - Projects Simple/projects/Anagram_Checker/Anagram_Checker.c
#include <stdio.h>
#include <string.h>
#include <ctype.h>

#define SIZE 256

int areAnagrams(char *str1, char *str2) {
    int count1[SIZE] = {0}, count2[SIZE] = {0};

    if (strlen(str1) != strlen(str2)) return 0;

    for (int i = 0; str1[i] && str2[i]; i++) {
        count1[tolower(str1[i])]++;
        count2[tolower(str2[i])]++;
    }

    for (int i = 0; i < SIZE; i++) {
        if (count1[i] != count2[i]) return 0;
    }
    return 1;
}

int main() {
    char str1[100], str2[100];
    
    printf("Enter first string: ");
    scanf("%s", str1);
    
    printf("Enter second string: ");
    scanf("%s", str2);

    if (areAnagrams(str1, str2)) 
        printf("The strings are anagrams.\n");
    else 
        printf("The strings are not anagrams.\n");

    return 0;
}
